packages/extension-base/src/stores/deprecated__Base.ts.ts

```diff
@@ -11,6 +11,9 @@ const lastError = (type: string): void => {
   }
 };
 
+/**
+ * @deprecated Use the utils/localStorageStores utility for local storage management.
```

Comment:
Because it's still used for the `Accounts` store, which I can't refactor because it's used with an external library that requires this `Base` store api.